pages/stylesheets/products.css

```diff
@@ -0,0 +1,157 @@
+.products-container{
+    margin-top: 9.4rem;
+    display: grid;
+    grid-template-columns: 1.2fr 4.6fr;
+    grid-template-areas: "aside main";
+    gap: 10px;
+}
+aside{
```

Comment:
Prehaps a class name could be added here, that way when you import your `products.css` into a main stylesheet, it will have less chances of overriding any css property which you might use for `aside` element